README.md

```diff
@@ -86,11 +87,13 @@ environment variable
 
 ##### Steps for Windows platform
 
-* Go to directory with cloned repository
+* Go to directory with cloned repository and enter subfolder tools\ms
 * Run setup_VS2013_x64.cmd to generate solution for Visual Studio 2013.
 **Note:** In case Visual Studio has been installed to non-default location -- modify path in .cmd file.
 **Note:** Double click on file or run from windows command prompt. DO NOT run in "git bash".
-* After script execution finish, go to build directory which is located one directory upper and open smartDeviceLinkCore.sln file using Visual Studio 2013
+* After script execution finish:
+  * Add path to the openssl executable to the PATH environment variable. Usually in build\openssl_win_x64\bin.
+  * Go to build directory which is located one directory upper and open smartDeviceLinkCore.sln file using Visual Studio 2013
```

Comment:
@pvvasilev thanks for the corrections!
Fixed in [f1215b4](https://github.com/smartdevicelink/sdl_core/pull/791/commits/f1215b46dafe9cf25b752c45288ae111a6d1c156)